### PR TITLE
Feature/add support CodeceptJS

### PR DESCRIPTION
**Pull request by CDJojn**
https://github.com/dixons/mobile-manager/issues/31

### CODE REVIEW

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+    {
+        /// <summary>
+        /// Initializes a new instance of the AndroidSeleniumConfig class.
+        /// </summary>
+        /// <param name="device">IDevice Device</param>
+        public AndroidSeleniumConfig(IDevice device) { 
+            this.Id = device.Id;
+            this.Name = device.Name;
+            this.AppiumEndpoint = device.AppiumEndpoint;
+            this.Host = !string.IsNullOrEmpty(device.AppiumEndpoint) ? new Uri(device.AppiumEndpoint).Host : "";
```
Comment (xfoggi) [inline MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: string.Empty would be better then ""

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+            this.Port = !string.IsNullOrEmpty(device.AppiumEndpoint) ? new Uri(device.AppiumEndpoint).Port.ToString() : "";
+        }
+
+        /// <inheritdoc />
+        /// <summary>
+        /// Gets or sets the Id.
+        /// </summary>
+        /// <value>The Id.</value>
+        public string Id { get; set; }
```
Comment (xfoggi) [inline MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: If you don't set properties anywhere (like only in constructor), you can omit the set to prohibit outside manipulation - same for ios config

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs [hunk 3]
```diff
@@ -0,0 +1,63 @@
+using System;
+using MobileManager.Models.Devices;
+using MobileManager.Models.Devices.Interfaces;
+using MobileManager.SeleniumConfigs.DataSets.Interfaces;
+using DotLiquid;
```
Comment (xfoggi) [inline MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: what is this and how it is used in this class?

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs [hunk 4]
```diff
@@ -0,0 +1,63 @@
...
+using MobileManager.SeleniumConfigs.DataSets.Interfaces;
+using DotLiquid;
+
+namespace MobileManager.SeleniumConfigs.DataSets
+{
+    /// <inheritdoc />
+    /// <summary>
+    /// AndroidSeleniumConfig.
+    /// </summary>
+    public class AndroidSeleniumConfig : Drop, ISeleniumConfig
```
Comment (xfoggi) [inline MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: what is "drop"?

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs (hunk 1 above)
Comment (CDJojn) [reply MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: Will be fixed

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs (hunk 2 above)
Comment (CDJojn) [reply MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: Make sense in this case

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs (hunk 3 above)
Comment (CDJojn) [reply MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: Templating system - https://github.com/dotliquid/dotliquid - .NET port of Liquid https://shopify.github.io/liquid/
I will add comments about this

File: MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs (hunk 4 above)
Comment (CDJojn) [reply MobileManager/SeleniumConfigs/DataSets/AndroidSeleniumConfig.cs]: Drop is part of Liquid technology - https://github.com/dotliquid/dotliquid/wiki/DotLiquid-Drops

**Pull request merged**